MoEPluginBase/Manager/MoEPluginController.m

```diff
@@ -0,0 +1,530 @@
+//
+//  MoEPluginController.m
+//  MoEPluginBase
+//
+//  Created by Rakshitha on 09/12/21.
+//
+
+#import <Foundation/Foundation.h>
+#import "MoEPluginController.h"
+#import "MoEPluginMessage.h"
+#import "MoEPluginUtils.h"
+#import "MoEngage/MoEngage.h"
+#import "MOInAppCampaign+Utility.h"
+#import "MoEPluginConstants.h"
+#import "MoEPluginMessageQueueHandler.h"
+#import "MoEngage/MoEngage.h"
+#import "MoEPluginCoordinator.h"
+#import <MoEngageObjCUtils/MoEngageObjCUtils.h>
+#import <MOEngageInApps/MOInApp.h>
+#import <MoEngageMessaging/MoEngageMessaging-Swift.h>
+#import "MOInAppSelfHandledCampaign+Utility.h"
+
+
+@interface MoEPluginController()<MOInAppNativDelegate, MOMessagingDelegate>
+@property(nonatomic, strong) MoEPluginMessageQueueHandler* messageHandler;
+
+
+@end
+
+@implementation MoEPluginController
+
+- (instancetype)init {
+    self.messageHandler = [[MoEPluginMessageQueueHandler alloc]init];
```

Comment:
Init is a method call. Should have space in between, which is missing.
`[[MoEPluginMessageQueueHandler alloc] init];`